### PR TITLE
Adds Keek pirate shit.

### PR DESCRIPTION
**Pull request by Qustinnus**
wow woopdiedoo its big daddy.

:cl: Keekenox and Floyd
imageadd: New pirate outfits and sprites
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 /obj/item/clothing/head/canada
 	name = "striped red tophat"
-	desc = " It feels sticky, like maple syrup - <i>il se sent collante, comme le sirop d'�rable</i>"
+	desc = " It feels sticky, like maple syrup - <i>il se sent collante, comme le sirop d'�rable</i>"
```
Comment (ninjanomnom) [inline code/modules/clothing/head/misc.dm]: You messed up a description here

Comment (Qustinnus) [inline code/modules/clothing/head/misc.dm]: didnt even touch it, this keeps happening in Atom whenever I touch a part of code with shitty text stuff like that.

**Comment by Qustinnus** [comment]
fixed

File: code/game/gamemodes/miniantags/ftl/enemy_ship/outfit/pirate.dm [hunk 2]
```diff
@@ -16,9 +18,8 @@
 /datum/outfit/defender/command/pirate
   name = "pirate ship captain"
   uniform = /obj/item/clothing/under/pirate
-  suit = /obj/item/clothing/suit/space/pirate
-  shoes = /obj/item/clothing/shoes/laceup
-  head = /obj/item/clothing/head/helmet/space/pirate
+  suit = /obj/item/clothing/suit/space/hardsuit/steampunk_pirate
+  head = /obj/item/clothing/head/helmet/space/hardsuit/steampunk_pirate
```
Comment (ninjanomnom) [inline code/game/gamemodes/miniantags/ftl/enemy_ship/outfit/pirate.dm]: Why does the captain have no shoes

Comment (Qustinnus) [inline code/game/gamemodes/miniantags/ftl/enemy_ship/outfit/pirate.dm]: my bad, i thought it was a child of defender/pirate.

**Pull request merged**